### ISSUE TITLE
Optimism Deployment: Aave v3

### ISSUE THREAD

**Issue by green-jeff** [vesperfi/vesper-pool-ops#238]
Deploy the following Aave v3 strategies to Optimism:

ETH: Deposit ETH borrow USDC deposit to Vesper

OP: Deposit OP borrow USDC deposit to Vesper

Note that OP is "isolated" which may require additional parameters. If it requires audit or is too cumbersome we may discuss deploying the strategy for just ETH pool.

**Comment by rokso** [comment]
yeah, we will have to come up with a good fix and not just remove `require` check

**Comment by virendrapatidar** [comment]
@rokso @patidarmanoj10 

Can we discuss how to proceed here?

**Comment by rokso** [comment]
we can do something like this for Vesper strategies in constructor
```
  address _poolRewards = vPool.poolRewards();
        if (_poolRewards != address(0)) {
        address _rewardToken =  IPoolRewards(_poolRewards).rewardTokens(0); // mostly it is VSP
        }
```

**Comment by rokso** [comment]
But for now, if you are blocked, may be solution is to remove `require` and deploy and put require back. But I am not sure having zero address as VSP will break something or not. If that break something then we may fix it properly.

**Comment by virendrapatidar** [comment]
PR is ready for review https://github.com/bloqpriv/vesper-contracts/pull/489

**Comment by virendrapatidar** [comment]
- Deployed AaveV3 strategy for vaETH pool
- Updated weight to 17%
- First rebalance done

**Comment by green-jeff** [comment]
@patidarmanoj10 @rokso 

Does our Aave strategy support harvesting of OP rewards on Vesper pool?

Reading above conversation, perhaps easy solution is to set vspAddress to OP token. This is assuming we receive OP grant and don't incentivize directly with VSP (right now, that is our thinking).
